Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
```

Comment:
The `VOLUME` instruction is missing the correct syntax. It should be `VOLUME ["/var/lib/mysql"]` to ensure the volume is correctly set up.